templates/common/infra/bicep/core/database/cosmos/mongo/cosmos-mongo-account.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'Creates a Cosmos for MongoDB Account.'
```

Comment:
Should this be "Creates an Azure Cosmos DB for Mongo instance" or is "account"correct here?

Reply:
"Azure Cosmos DB for MongoDB account" - https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/how-to-configure-capabilities.